### PR TITLE
Add example for sync jobs to docs

### PR DESCRIPTION
**Pull request by cmangla**
See issue #230

### CODE REVIEW

File: docs/examples/sync_job.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def sync_task(t):
```
Comment (samuelcolvin) [inline docs/examples/sync_job.py:1]: please include all imports required to run the example.

**Review by samuelcolvin — COMMENTED** [review]
otherwise looks good to me.

File: docs/examples/sync_job.py (hunk 1 above)
Comment (cmangla) [reply docs/examples/sync_job.py:6]: Done.

**Comment by samuelcolvin** [comment]
Thanks so much.

**Pull request merged**